setup.py

```diff
@@ -100,12 +99,16 @@ def run(self):
 hdfs = ["pyarrow>=2.0.0;  python_version < '3.9'"]
 webhdfs = ["hdfs==2.5.8"]
 webdav = ["webdavclient3>=3.14.5"]
+osf = ["osfclient==0.0.4"]
```

Comment:
@jorgeorpinel, this is what makes `dvc[osf]` works. So, it's optional. The below one is for the tests.